app/components/StudioEditor.vue.ts

```diff
@@ -358,6 +359,16 @@ export default class StudioEditor extends TsxComponent {
       rect.crop,
       { x: rect.x, y: rect.y },
     );
+
+    // set the game_capture's auto resize to false
+    if (source.type === 'game_capture') {
+      this.editorCommandsService.executeCommand('EditSourcePropertiesCommand', source.id, [
```

Comment:
This shouldn't be done as a command.  This will result in a new entry in the undo history, which will not combine with the command above it.  This breaks the "rule" of a single user action resulting in a single new entry in the undo history.  I think unfortunately we will need to roll this behavior into the `CropItemsCommand` if it is required whenever we crop the game capture.